src/spel2_dll/spel2.cpp

```diff
@@ -24,12 +24,7 @@ void InitSoundManager(Spelunky_DecodeAudioFile* decode_function)
         [](const char* file_path)
         {
             Spelunky_DecodedAudioBuffer buffer = local_decode_function(file_path);
-            return DecodedAudioBuffer{
-                buffer.num_channels,
-                buffer.frequency,
-                static_cast<SoundFormat>(buffer.format),
-                std::unique_ptr<const std::byte[]>{reinterpret_cast<const std::byte*>(buffer.data)},
-                buffer.data_size};
+            return DecodedAudioBuffer{buffer.num_channels, buffer.frequency, static_cast<SoundFormat>(buffer.format), std::unique_ptr<const std::byte[]>{reinterpret_cast<const std::byte*>(buffer.data)}, buffer.data_size};
```

Comment:
Can we add trailing comma on the three in this file? they seemed more readable broken over many lines